File: fixtures/Calculator.java
public class Calculator {
    public int add(int a, int b) {
        if (a == 0 || b == 0) {
            return 0;
        }
        return a + b;
    }
    
    public int multiply(int x, int y) {
        if (x == 0 || y == 0) {
            return 0;
        }
        return x * y;
    }
}

File: fixtures/calculator.ts
interface MathOperations {
  add(a: number, b: number): number;
  multiply(a: number, b: number): number;
}

class Calculator implements MathOperations {
  add(a: number, b: number): number {
    if (a === null || b === null) {
      return 0;
    }
    return a + b;
  }
  
  multiply(a: number, b: number): number {
    if (a === null || b === null) {
      return 0;
    }
    return a * b;
  }
}